PWGHF/HFC/Utils/utilsCorrelations.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file utilsCorrelations.h
+/// \brief Xu Wang <waxu@cern.ch>
+
+#ifndef PWGHF_HFC_UTILS_UTILSCORRELATIONS_H_
+#define PWGHF_HFC_UTILS_UTILSCORRELATIONS_H_
+
+#include <cmath>
```

Comment:
Fix

Reply:
I'm sorry, but I don't quite understand what you mean by "fix."

Reply:
You have not addressed my previous comments correctly. See https://github.com/AliceO2Group/O2Physics/pull/8183#discussion_r1820613158

Reply:
Sorry, I overlooked this message.

Reply:
Sorry, I overlooked this message.

Reply:
The header with math constants is still missing.